dbt/scripts/export_qc_town_close_reports.py

```diff
@@ -87,7 +87,7 @@ def is_tri(township_code: str, year: int) -> bool:
         raise ValueError(f"'{township_code}' is not a valid township code")
     # 2024 is the City reassessment year (tri code 1), so
     # ((2024 - 2024) % 3) + 1 == 1, and so on for the other two tris
-    return str((year - 2024 % 3) + 1) == tri
```

Comment:
Python [prioritizes modulo over subtraction in its order of operations](https://textbooks.cs.ksu.edu/intro-python/02-numbers/07-order-of-operations/), so this expression was previously always simplifying to `(year - 2) + 1`, which only matches the possible `tri` values of 1-3 for `year` values on the range `[2, 4]`. As a result, all towns were returning `False` for `is_tri()` in 2024, which I noticed while running the Jefferson report.

Reply:
Damn 🤦 I should've caught this in review. Nice catch.